### PR TITLE
add helpers file, create function to format cids lenght

### PR DESCRIPTION
**Pull request by ionutiorgulescu**

### CODE REVIEW

File: src/shared/helpers.js [hunk 1]
```diff
@@ -0,0 +1,7 @@
+export const formatCid = (cid) => {
```
Comment (brunolm) [inline src/shared/helpers.js]: I'd rename the file from `helpers.js` to `formatCID.js`

Comment (ionutiorgulescu) [reply src/shared/helpers.js:1]: This is a helper method, maybe we'll have other helpers methods? We'll create for each method a new file?

**Comment by ionutiorgulescu** [comment]
image

**Comment by ghost** [comment]
Cherry picked this over to the release branch and worked well for me.

LGTM and merging now.

**Pull request merged**